models/olids/base/base_olids_flag.sql

```
@@ -35,6 +35,6 @@ SELECT
 FROM {{ source('olids_common', 'FLAG') }} src
 INNER JOIN {{ ref('base_olids_patient') }} patients
     ON src.patient_id = patients.id
-INNER JOIN {{ ref('int_ncl_practices') }} ncl_practices
-    ON src.record_owner_organisation_code = ncl_practices.practice_code
+INNER JOIN {{ ref('int_wnl_practices') }} wnl_practices
```

SQLFluff (4.1.0)

[error] Implicit/explicit aliasing of table.

(AL01)